src/ChainClaim.sol

```diff
@@ -67,11 +72,12 @@ abstract contract ChainClaim is EIP712 {
   function isValidClaimantSig(
     address issuedAddress,
     address destinationAddress,
```

Comment:
```suggestion
```

Reply:
What's the context for this suggestion? The destination address is signed by the issuer and we'd need to use it's hash to recover the issuer address

Reply:
I suggest to remove it as it isn't used below or did I miss smth?

Reply:
my bad it is used